systems/odo_user_guide.rst

```diff
@@ -0,0 +1,147 @@
+.. _odo-user-guide:
+
+******
+Odo
+******
+
+System Overview
+===============
+
+.. note::
+    Odo is a training system that is not intended to be used as
+    an OLCF user resource. Access to the system is only obtained through
+    OLCF training events.
+
+Odo is a 32-node stand-alone system with the same architecture as
+Frontier,so most of the Frontier User Guide can be referenced for
```

Comment:
Can we add a hyperlink to the Frontier Guide when saying "Frontier User Guide" here?